app/src/main/java/com/example/seonoh/seonohapp/CoinAdapter.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.seonoh.seonohapp
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.seonoh.seonohapp.model.CurrentPriceInfoModel
+
+class CoinAdapter(data: ArrayList<CurrentPriceInfoModel>,val type : Int) : RecyclerView.Adapter<KrwItemViewHolder>(){
+
+    lateinit var mView : View
```

Comment:
```suggestion
    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): KrwItemViewHolder {
        val mView = LayoutInflater.from(parent.context).inflate(R.layout.krw_item,parent,false)
        return KrwItemViewHolder(mView,type)
    }
```
mView는 onCreateViewHolder 함수 내에서만 사용하므로 함수의 멤버 변수로 선언하는 것이 좋을 것 같습니다

Reply:
네, mView 멤버 변수로 변경했습니다!